ecosystem/sep-0024.md

```diff
@@ -107,6 +109,78 @@ non-equivalent asset transactions, specifically `amount_in_asset`, `amount_fee_a
 allows client applications can relay this information, in addition to the amounts specified in the associated
 attributes, back to their users.
 
+Alternatively, the Anchor can support the [SEP-38] RFQ API and the `quote_id` attribute in
+`POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive` requests. This enables wallets to
+check the Anchor's exchange rates prior to initiating a transaction and obtain a firm quote to use for an upcoming
+transaction.
+
+### Asset Conversions Using the Market Rate
+
+To convert at the market rate, clients get an
+[indicative quote](https://www.investopedia.com/terms/i/indicativequote.asp) with [SEP-38] and then proceed to calling
+the [`POST /transactions/withdraw/interactive`](#withdraw) or [`POST /transactions/deposit/interactive`](#deposit)
+endpoints. Market rate conversions executed in this way do not provide a guarantee of the final price to be paid for the
+asset. Here is how this kind of conversion should happen in deposits and withdrawals for an asset pair like
+`BRL <> USDC`:
+
+Anchors offer indicative quote support via enabling `indicative_quotes_supported` feature flag.
+
+#### Deposit:
```

Comment:
I am not sure if this is an anti-pattern of the interactive UI flow. This would make sense in SEP-6. But for SEP-24, why is the reason not to let the anchor finalize the quote in the interactive UI?

Reply:
They can finalize it in interactive UI, however wallet may want to fetch the firm quote and agree with anchor to use that particular quote.
FYI: in this protocol suggestion this functionality is optional. Anchor may only support indicate qoutes